docs/index.html

```diff
@@ -43,10 +66,26 @@
       markdown: {
         renderer: {
           code: function(code, lang) {
-            if (lang === "mermaid") {
-              return (
-                '<div class="mermaid">' + mermaid.render('mermaid-svg-' + num++, code) + "</div>"
-              );
+            if (lang.startsWith('mermaid')) {
+              var resultingHTML = '';
+              
+              if (lang === "mermaid-code" || lang === 'mermaid-example') {
```

Comment:
Should we [change this](https://github.com/mermaid-js/mermaid/pull/2449#discussion_r741306869) to `mmd`?